### ISSUE TITLE
evalute

### ISSUE THREAD

**Issue by pavelxx1** [TR-tryingnewstuff/RLlib_position_intraday#1]
Hi!
Why in backtesting you use .train() method ?

**Comment by TR-tryingnewstuff** [comment]
No real thought process behind this choice, I saw that it worked and am not familiar enough with the ray library to know whether there is a more suitable approach but if you do I'd love to hear your recommendation ! 
NB: There may be some 'dumb' mistakes here and there as I like to try lots of different things so sometimes I forget to change all that is needed.

**Comment by pavelxx1** [comment]
I will test your code, and if there are any issues, I will let you know.

**Comment by pavelxx1** [comment]
Why in custom_model.Conv2d u didn't use any activation function, such 'relu' ?

**Comment by TR-tryingnewstuff** [comment]
My mistake, I use the tanh activation function in my other files as I think the relu is not fully able to capture what I am trying to predict (as values range from 0 to inf contrary to tanh where it ranges from -1, 1), though you can try both and see what gives the best results.

**Comment by pavelxx1** [comment]
For agent testing u can use this simple code template
```
def test(path):
    fym.config.update({
        "config.env_config.max_t": 50,
    })
    env_config = fym.config.load("config.env_config", as_dict=True)

    agent = ppo.PPOTrainer(env=MyEnv, config={"explore": False})
    agent.restore(path)

    env = MyEnv(env_config)
    env.logger = fym.Logger("data.h5")

    # run until episode ends
    done = False
    obs = env.reset()
    while not done:
        env.render()
        action = agent.compute_single_action(obs)
        obs, reward, done, info = env.step(action)

    env.close()

```
u can improve:
- add regularization for model and fcn layer
- normalize image
- instead relu u can use gelu it can process neurons with negative value if need
